large_job_submission.py

```diff
@@ -0,0 +1,90 @@
+"""Submits simulations to the API."""
+import os
+
+import numpy as np
+
+from lib import models
+from lib import scenarios
+
+import inductiva
+
+INPUT_DATASET = "Path/To/Dir/With/Meshses"
+WORKING_DIR = None
+
+FLOW_VELOCITY_RANGE_X = [20, 30]
+FLOW_VELOCITY_RANGE_Y = [0, 0]
+FLOW_VELOCITY_RANGE_Z = [0, 0]
+NUM_SIMULATIONS_PER_OBJECT = 1
+
+DOMAIN_X = [-5, 20]
+DOMAIN_Y = [-5, 5]
+DOMAIN_Z = [0, 8]
+NUM_ITERATIONS = 100
+RESOLUTION = 2
+
+MACHINE_TYPE = "c2-standard-16"
+NUM_MACHINES = 1
+DISK_SIZE_GB = 70
+ELASTIC_MACHINE_GROUP = True
```

Comment:
I think we can still reduce entropy and avoid setting elastic machine groups for now. In this repo the goal is to validate the API for large scale submissions. The MachineGroup is enough for that. In the ReadMe we can point people to that, but let's remove this setting from here at the moment.

Reply:
Agree